src/Controller/AuditController.php

```diff
@@ -36,7 +36,7 @@ public function indexAction($page = 1)
         $reader = $this->getAuditReader();
         $revisions = $reader->findRevisionHistory(20, 20 * ($page - 1));
 
-        return $this->render('SimpleThingsEntityAuditBundle:Audit:index.html.twig', [
+        return $this->render('@SimpleThingsEntityAudit\Audit\index.html.twig', [
```

Comment:
Why are we using backslashes?

Reply:
🤦🏻 I do it again. I dont know, mybe I love backslashes. Fixed.